### DISCUSSION TITLE
is there any way to deploy superset on kubernetes without helm charts? i want to deploy my forked superset version on kubernetes

### DISCUSSION THREAD

**Discussion by rahularram1999** [apache/superset#18889] [Q&A / Help]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by srinify** [comment]
Hey @rahularram1999 since this isn't a bug, I'm moving this over to Discussions :)

**Comment by HaoXuAI** [comment]
Any updates on this?
